.circleci/config.yml

```diff
@@ -148,4 +202,14 @@ workflows:
           filters:
             branches:
               only:
-                - master
+                - igor/nightly-build
```

Comment:
Why is this changed?

Reply:
I was getting an error to persist a temporary file to the `pushdoc` job.
Changed back to master.